lab_1/main.py

```diff
@@ -27,6 +61,19 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     """
     pass
 
+    if not isinstance(tokens, list):
+        return []
+    if not isinstance(stop_words, list):
+        return tokens
+
+    for word in stop_words:
+        while word in tokens:
+            tokens.remove(word)
+    return tokens
+
+
+tokens = remove_stop_words(tokens, stop_words)
```

Comment:
There should be no function calling\using in `main.py` -> use functions on the`start.py`.